recipes/stereo-pipeline/recipe.yaml

```diff
@@ -0,0 +1,154 @@
+context:
+  name: stereo-pipeline
+  version: "3.5.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+    # Main source code
+    - url: https://github.com/NeoGeographyToolkit/StereoPipeline/releases/download/${{ version }}/StereoPipeline.tar.gz
+      sha256: ccb192b1c6e7a6c9b72719c88a407988490ecfc4c8cbae97fc1811225c160f84
+      target_directory: StereoPipeline
+    
+    # Submodules
+    
+    - url: https://github.com/NeoGeographyToolkit/StereoPipeline/releases/download/${{ version }}/MultiView.tar.gz
+      sha256: d69d34adc263dceb023bbfd0910c3ab0e69a3dc67292c046860e924e21a2658a
+      target_directory: MultiView
+
+    - url: https://github.com/NeoGeographyToolkit/StereoPipeline/releases/download/geoid1.0/geoids.tgz
+      sha256: 2cffdac6efbfa47a94e8adb650489a267de72770f06656df35f37a531d999f5d
+      target_directory: geoids
+    
+    - url: https://github.com/NeoGeographyToolkit/libnabo/archive/refs/tags/3.5.0.zip
+      sha256: 4b260a5ff3fc6fff56ff500d2b14c1cc4136ac58a3ed306e78dee8769261ae3c
+      target_directory: libnabo
+
+    - url: https://github.com/NeoGeographyToolkit/libpointmatcher/archive/refs/tags/3.5.0.zip
+      sha256: 75f58c416d98691f0c88cd5eb712dfaeb2fb478d1c21a14065ed6c17264310f8
+      target_directory: libpointmatcher
+      
+    - url: https://github.com/NeoGeographyToolkit/FastGlobalRegistration/archive/refs/tags/3.5.0.zip
+      sha256: 3bd48acc05af13fc5dfbf4ca397f9ffa28fd53c83a28346358dff0726e4d2996
+      target_directory: FastGlobalRegistration
+      
+    - url: https://github.com/NeoGeographyToolkit/s2p/releases/download/3.5.0/s2p.tar.gz
+      sha256: 8752edac97f9596266d2712123ea1d14e44e7c201c8ea4062223708886ae5237
+      target_directory: s2p
+      
+    - url: https://github.com/NeoGeographyToolkit/libelas/archive/refs/tags/3.5.0.zip
+      sha256: d160bbfd020aefac4782a5b269d9c0f8d6af5f43cb8951d27204d2068315521e
+      target_directory: libelas
```

Comment:
All the hardcoded versions here will make this impossible for the bot and our infrastructure to handle this recipe.

Why not separate packages in separate recipes?

Reply:
I can prepare one single code archive, with a single sha256, packaging these into subdirectories. conda-forge won't care then.

Reply:
I addressed I think the biggest sticking point here. I have now a single versioned source package.